### PR TITLE
[FLINK-10102][docs] Fix docs for EXECUTION_FAILOVER_STRATEGY

### PR DESCRIPTION
**Pull request by maqingxiang**

### CODE REVIEW

**Comment by zentol** [comment]
If you modify the description of config options you have to regenerate the documentation. See the test error message for instructions.

**Comment by dawidwys** [comment]
@maqingxiang could you regenerate the documentation?

**Comment by zentol** [comment]
I'm taking over the PR. @dawidwys could you take a look? I've rebased the PR, reworked the description and regenerated the docs.

File: flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java [hunk 1]
```diff
@@ -102,12 +104,21 @@
...
 	 */
 	public static final ConfigOption<String> EXECUTION_FAILOVER_STRATEGY =
 		key("jobmanager.execution.failover-strategy")
 			.defaultValue("full")
-			.withDescription("The maximum number of prior execution attempts kept in history.");
+			.withDescription(Description.builder()
+				.text("This options specifies the job computation recovers from task failures.")
+				.linebreak()
+				.text("Accepted values are:")
+				.linebreak()
```
Comment (dawidwys) [inline flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java]: I think lists always render in a separate line. So I would say this `linebreak` is unnecessary. The linebreak before `Accepted values are:` is also questionable right now. It renders like this:

screenshot from 2018-08-07 15-46-16

I would either remove the line break or add two linebreaks to render empty line.

File: flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java [hunk 2]
```diff
@@ -102,12 +104,21 @@
...
+				.list(
+					text("'full': Restarts all tasks."),
+					text("'individual': Restarts only the failed task. Should only be used if all tasks are independent components."),
+					text("'region': Restarts all tasks that could be affected the task failure.")
```
Comment (dawidwys) [inline flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java]: affected **by** the task failure

**Comment by maqingxiang** [comment]
I'm very sorry for the late reply. Thank you @zentol  and @dawidwys

**Pull request merged**